sympy/core/relational.py

```diff
@@ -388,6 +392,38 @@ def __new__(cls, lhs, rhs=0, **options):
     def _eval_relation(cls, lhs, rhs):
         return _sympify(lhs == rhs)
 
+    def _eval_rewrite_as_Add(self, *args, **kwargs):
+        """return Eq(L, R) as L - R. To control the evaluation of
+        the result set pass `evaluate=True` to give L - R;
+        if `evaluate=None` then terms in L and R will not cancel
+        but they will be listed in canonical order; otherwise
+        non-canonical args will be returned.
+
+        Examples
+        ========
+
+        >>> from sympy import Eq, Add
+        >>> from sympy.abc import b, x
+        >>> eq = Eq(x + b, x - b)
+        >>> eq.rewrite(Add)
+        2*b
+        >>> eq.rewrite(Add, evaluate=None).args
+        (b, b, x, -x)
+        >>> eq.rewrite(Add, evaluate=False).args
+        (b, x, b, -x)
+        """
+        L, R = args
+        eval = kwargs.get('evaluate', True)
```

Comment:
Minor nit: better to call this variable `evaluate` instead of `eval`, since `eval` is a builtin name (which would never be used in this function to be sure, but it could also confuse code that checks for `eval` usage since it's a dangerous function).

Reply:
done